api/v1alpha1/oadp_types.go

```diff
@@ -172,12 +172,17 @@ type DataMover struct {
 	// +optional
 	CredentialName string `json:"credentialName,omitempty"`
 	// User supplied timeout to be used for VolumeSnapshotBackup and VolumeSnapshotRestore to complete, default value is 10m
+	// +kubebuilder:default="10m"
 	// +optional
 	Timeout string `json:"timeout,omitempty"`
 	// the number of batched volumeSnapshotBackups that can be inProgress at once, default value is 10
+	// +kubebuilder:default=10
+	// +kubebuilder:validation:Minimum=1
```

Comment:
Can we change this minimum to 0? We want to use this value as "unlimited" and will document it afterwards.